### ISSUE TITLE
App shows two actions instead of the expected one.

### ISSUE THREAD

**Issue by SireInsectus** [brbeaird/SmartThings_MyQ#53]
I'm seeing the option **Door control** and a second one **Garage Door Control** which also has an open/closed status.

**Door control** seems to open the door without any "warnings" but can never close it (see previous error about inability to connect)

**Garage Door Control** seems to open without warning, but close with a lot of flashing light and alarms (which is probably good).

The key problem, is that *Door control** appears to be invoked from the tile which does not work properly - shouldn't there be only one option here?

**Comment by brbeaird** [comment]
Yeah I’m gonna drop Door Control today and do some testing. Don’t think we
need both.

**Comment by brbeaird** [comment]
Just pushed an update that removes that door control.  See if that works better for you.

**Comment by brbeaird** [comment]
FYI - I had to revert this change. Unfortunately, several older apps/automations rely on the Door Control capability, so we're stuck with it for now.
